python/tests/test_eclfile.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+import sys
+import numpy as np
+
+from opm.io.ecl import EclFile
+
+
+class TestEclFile(unittest.TestCase):
+    
+    def test_getListOfArrays(self):
+
+        refList=["INTEHEAD","LOGIHEAD","DOUBHEAD","PORV","DEPTH","DX","DY","DZ","PORO",
+                 "PERMX","PERMY", "PERMZ","NTG","TRANX","TRANY","TRANZ","TABDIMS","TAB",
+                 "ACTNUM","EQLNUM","FIPNUM","PVTNUM","SATNUM","TRANNNC"]
+    
+        self.assertRaises(ValueError, EclFile, "/file/that/does_not_exists")
+
+        file2uf = EclFile("tests/data/SPE9.INIT")
+        self.assertEqual(file2uf.getNumArrays(), 24)
+
+        arrList = file2uf.getListOfArrays()
+        self.assertEqual(arrList, refList)
+
+        file2f = EclFile("tests/data/SPE9.FINIT")
```

Comment:
No reason to add the formatted testdata - formatted reading is already tested in the underlying C++ class